src/styles/blocks/day.scss

```diff
@@ -0,0 +1,36 @@
+.day {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  width: $sizeDay;
+  height: $sizeDay;
+  margin: $between;
+  background-color: #eee;
+  border: 1px solid #000;
+  transition-duration: 0.5s;
+  transition-property: transform background-color;
+
+  @for $index from 1 through 31 {
+    &:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  @each $name, $day in $days {
+    &--day--#{$name} {
+      margin-left: $day;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &--mounth--#{$index} {
+      display: flex;
+    }
+  }
```

Comment:
why do you need it here? just add display: flex to common class `.calendar__day`

![image](https://user-images.githubusercontent.com/57598825/113678019-dbc8e800-96c6-11eb-9a55-457a3b6094c1.png)